dll/win32/shell32/utils.cpp

```diff
@@ -9,6 +9,46 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL(shell);
 
+/*************************************************************************
+ *                SHGetShellStyleHInstance (SHELL32.749)
+ */
+EXTERN_C HINSTANCE WINAPI SHGetShellStyleHInstance(VOID)
+{
+    HINSTANCE hInst = NULL;
+    WCHAR szPath[MAX_PATH], szColorName[100];
+    HRESULT hr;
+    CStringW strShellStyle;
+
+    /* First, attempt to load the shellstyle dll from the current active theme */
+    TRACE("\n");
+
```

Comment:
```suggestion

    TRACE("\n");

    /* First, attempt to load the shellstyle dll from the current active theme */
```
Not sure the TRACE helps in any case.